MMM-nyc-transit.js

```diff
@@ -323,12 +323,28 @@ Module.register('MMM-nyc-transit', { /*eslint-disable-line*/
     }, loadTime)
   },
 
+  arrayEquals: function (a, b) {
```

Comment:
@joolean, do you think we should be explicit in the function signature names? It will make it easier to read and understand. It is more verbose, but then you/we are documenting in the signature vs comments and or jsDocs.

e.g. `arrayEquals: function(payloadStaions, myStations)` or some variant other than `arrayEquals: function(a, b)`

Reply:
@joolean; Have you had a chance to read my review/comment?

Reply:
Yep! Was busy with work. Just pushed a change.